src/tests/tracing/runtimeeventsource/NativeRuntimeLevelKeywords.cs

```diff
@@ -0,0 +1,136 @@
+﻿using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Diagnostics.Tracing;
+using System.Linq;
+
+using System.Buffers; // Copied from https://github.com/dotnet/runtime/pull/86370 by @davmason:
+{
+    // Access ArrayPool.Shared.Rent() before the test to avoid the deadlock reported
+    // in https://github.com/dotnet/runtime/issues/86233. This is a real issue,
+    // but only seen if you have a short lived EventListener and create EventSources
+    // in your OnEventWritten callback so we don't expect customers to hit it.
+    byte[] localBuffer = ArrayPool<byte>.Shared.Rent(10);
+    Console.WriteLine($"buffer length={localBuffer.Length}");
+}
+Console.WriteLine();
+
+using GCListener informational = GCListener.StartNew(EventLevel.Informational);
+using GCListener verbose = GCListener.StartNew(EventLevel.Verbose);
+using GCListener logAlways = GCListener.StartNew(EventLevel.LogAlways);
+
+Stopwatch stopwatch = Stopwatch.StartNew();
+do
+{
+    int nanoseconds = (int)double.Min(stopwatch.Elapsed.TotalNanoseconds, 1000000d);
+
+    for (int i = 0; i < nanoseconds; i++)
+    {
+        _ = new object();
+    }
+    GC.Collect();
+
+    if (informational.Contains("GCStart_V2") && verbose.Contains("GCAllocationTick_V4"))
```

Comment:
Maybe you should pick some events that is supported by Mono runtime? The GCStart_V2 is supported when requesting full GC dumps (but currently won't be emitted if you just use event mask 1) and GCAllocationTick_V4 is not supported at all on Mono runtime. https://github.com/dotnet/runtime/blob/main/src/mono/mono/eventpipe/gen-eventing-event-inc.lst lists the native runtime event currently supported by Mono runtime. Mono runtime also supports all managed events emitted through EventSource implementations in BCL (and custom defined EventSource classes). If you don't find any suitable events around supported once, you will need to exclude this test on Mono runtime altogether, but would be really good to test it on Mono runtime as well. One option could be to request a GC dump and collect generated events from that, see https://github.com/dotnet/runtime/blob/main/src/tests/tracing/eventpipe/gcdump/gcdump.cs, that test runs on Mono desktop platforms.